src/Symfony/Component/Form/Extension/Core/Type/TimeType.php

```diff
@@ -210,6 +210,20 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 new DateTimeToArrayTransformer($options['model_timezone'], $options['model_timezone'], $parts, 'text' === $options['widget'], $options['reference_date'])
             ));
         }
+
+        if (\in_array($options['input'], ['datetime', 'datetime_immutable'], true) && null !== $options['model_timezone']) {
+            $builder->addEventListener(FormEvents::PRE_SET_DATA, static function (FormEvent $event) use ($options): void {
+                $date = $event->getData();
+
+                if (!$date instanceof \DateTimeInterface) {
+                    return;
+                }
+
+                if ($date->getTimezone()->getName() !== $options['model_timezone']) {
+                    trigger_deprecation('symfony/form', '6.2', sprintf('Using a %s instance with a timezone ("%s") not matching the configured model timezone "%s" is deprecated.', \get_class($date), $date->getTimezone()->getName(), $options['model_timezone']));
```

Comment:
```suggestion
                    trigger_deprecation('symfony/form', '6.2', sprintf('Using a "%s" instance with a timezone ("%s") not matching the configured model timezone "%s" is deprecated.', \get_class($date), $date->getTimezone()->getName(), $options['model_timezone']));
```
?
Should it mention that it won't be support and/or trigger an exception, or being implicit is enough?